cfy_manager/components/restservice/scripts/create_tables_and_add_defaults.py

```diff
@@ -157,6 +153,39 @@ def _add_provider_context(context):
     sm.put(provider_context)
 
 
+def _add_manager_status_reporter_user():
+    print('Creating the Manager Status Reporter user, default tenant and '
+          'security roles')
+    user = storage_utils.create_status_reporter_user_and_assign_role(
+        username=script_config['manager_status_reporter_username'],
+        password=script_config['manager_status_reporter_password'],
+        role=script_config['status_reporter_role'],
```

Comment:
i'm not sure why this has to be parameterized

Reply:
Actually its pretty clear what are the parameters, you're right.